srcpkgs/libnma/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'libnma'
+pkgname=libnma
+version=1.8.30
+revision=1
+build_style=meson
+build_helper="gir"
+hostmakedepends="cmake gettext glib-devel gtk-doc pkg-config vala"
```

Comment:
are you sure you need cmake given this is a meson-built package?

Reply:
When I was testing building the package locally I needed it. However, after my most recent test it seems like I don't. I've removed it from my most recent push and it hasn't introduced new errors.